### PR TITLE
Implement ACM certificates resource

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: resources/acm_certificates.go [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+				Name:        "renewal_summary_updated_at",
+				Description: "The time at which the renewal summary was last updated.  This member is required.",
+				Type:        schema.TypeTimestamp,
+				Resolver:    schema.PathResolver("RenewalSummary.UpdatedAt"),
+			},
+			{
+				Name:        "renewal_summary_renewal_status_reason",
+				Description: "The reason that a renewal request was unsuccessful.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("RenewalSummary.RenewalStatusReason"),
```
Comment (roneli) [inline resources/acm_certificates.go:153]: remove duplicate renewal?  i.e `renewal_summary_status_reason`

File: resources/acm_certificates.go [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+func AcmCertificates() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_acm_certificates",
+		Description:  "Contains metadata about an ACM certificate",
+		Resolver:     fetchAcmCertificates,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Columns: []schema.Column{
```
Comment (roneli) [inline resources/acm_certificates.go:23]: missing PK

**Review by roneli — CHANGES_REQUESTED** [review]
Some review comments

File: resources/acm_certificates.go (hunk 1 above)
Comment (irmatov) [reply resources/acm_certificates.go:153]: Where's the duplicate? Do you mean `renewal_summary_renewal_status` and `renewal_summary_renewal_status_reason` ? They are different, first one is the actual renewal status, and the second one is *failiure* reason (if any, I guess). I'll rename it to `renewal_summary_failure_reason`.

File: resources/acm_certificates.go (hunk 2 above)
Comment (irmatov) [reply resources/acm_certificates.go:23]: Good catch, added.

File: resources/acm_certificates.go [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+				Type:        schema.TypeJSON,
+				Resolver: resolveACMCertificateJSONField(func(cd *types.CertificateDetail) interface{} {
+					if cd.RenewalSummary == nil {
+						return nil
+					}
+					return cd.RenewalSummary.DomainValidationOptions
+				}),
+			},
+			{
+				Name:        "renewal_summary_renewal_status",
```
Comment (roneli) [inline resources/acm_certificates.go:139]:
```suggestion
				Name:        "renewal_summary_status",
```

File: resources/acm_certificates.go (hunk 1 above)
Comment (roneli) [reply resources/acm_certificates.go]: I meant the renaming of the following columns, sorry if it wasn't clear.

**Review by roneli — CHANGES_REQUESTED** [review]
Would add https://docs.aws.amazon.com/acm/latest/userguide/sdk-listtag.html list tags for certificate

**Comment by irmatov** [comment]
Added tags

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**